packages/cli/src/__tests__/commands/config/reset.test.ts

```diff
@@ -14,6 +14,8 @@ describe('ConfigResetCommand', () => {
 
 	const resetManagedConfigMock = jest.mocked(resetManagedConfig)
 
+	const profileName = process.env.SMARTTHINGS_PROFILE ?? 'default'
```

Comment:
It would be more "unit"y to not even use the environment variable at all but rather to mock it. Probably the simplest way to mock it (that I can think of) would be to spy on the `profileName` `get` method of the `SmartThingsCommand` prototype:

```
const profileNameSpy = jest.spyOn(SmartThingsCommand.prototype, 'profileName', 'get')
profileNameSpy.mockReturnValue('default')
```

Since spies in jest don't reset the same as everything else, you have to call `mockRestore` on the spy when you're done with it so it doesn't interfere with other methods.

Reply:
Would it work to instead do `profileNameSpy.mockReturnValueOnce('default')`, or does that not work with spies?

Reply:
It turns out `mockReturnValueOnce` without a `mockRestore()` does not work, but your solution does.